Packs/CortexXDR/Integrations/XDR_iocs/XDR_iocs.py

```diff
@@ -48,9 +48,7 @@ def __init__(self, params: Dict):
         self._base_url: str = urljoin(params.get('url'), '/public_api/v1/indicators/')
         self._verify_cert: bool = not params.get('insecure', False)
         self._headers: Dict = get_headers(params)
-        self._proxy = params.get('proxy', False)
-        if self._proxy:
-            self._proxy = handle_proxy()
+        self._proxy = handle_proxy()
```

Comment:
just run `handle_proxy` in the main func, no need to assign it to a attribute as its not used